src/components/FactionButton.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {TouchableOpacity, Image, View, Text, StyleSheet} from 'react-native';
+
+export const FactionButton = ({image, label, borderColor}) => {
+  return (
+    <TouchableOpacity style={[styles.imgFac, {borderColor}]}>
```

Comment:
Missing one param, "isSelected" to change the style of the button when it's true